do

```diff
@@ -46,8 +46,8 @@ images() {
 help_images_for_server="Build and push the Docker images and manifests for supported server versions."
 images-for-server() {
     MAJOR_SERVER_VERSION=4
-    MINOR_VERSION_START=4
-    MINOR_VERSION_END=8
+    MINOR_VERSION_START=9
+    MINOR_VERSION_END=0
```

Comment:
Shouldn't this be 7 and 9? Since we support the latest two versions, plus the RC 4.9.